### PR TITLE
feat: management workspace customization

### PR DESCRIPTION
**Pull request by yuye-aws**
### Description

1. Register **Dev tools** in OpenSearch Features
2. Add **Dev tools** and **Data sources** to management workspace.
3. Remove special logic for public and management workspaces.

## Screenshot

image

### Check List

- [ ] All tests pass
  - [x] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/workspace/server/plugin.ts [hunk 1]
```diff
@@ -155,7 +156,8 @@ export class WorkspacePlugin implements Plugin<{}, {}> {
           features: [
             `@${DEFAULT_APP_CATEGORIES.management.id}`,
             WORKSPACE_OVERVIEW_APP_ID,
-            WORKSPACE_UPDATE_APP_ID,
```
Comment (ruanyl) [inline src/plugins/workspace/server/plugin.ts:158]: I think `WORKSPACE_UPDATE_APP_ID` should not be removed, we should allow people to update management workspace settings

Comment (yuye-aws) [reply src/plugins/workspace/server/plugin.ts:158]: Should **Workspace settings** be removed for our initial release?

**Review by ruanyl — APPROVED** [review]

**Review by Hailong-am — APPROVED** [review]

**Review by wanglam — APPROVED** [review]

**Comment by ruanyl** [comment]
Why this is closed?

**Comment by Hailong-am** [comment]
> Why this is closed?

I guess it is by mistake, `yuye-aws deleted the reserved_workspace branch [19 minutes ago]` that will auto close the PR

**Comment by yuye-aws** [comment]
Sorry, I thought this branch has been merged

**Pull request merged**